src/NServiceBus.Core/NServiceBus.Core.csproj

```diff
@@ -13,7 +13,6 @@
   <ItemGroup Condition="'$(TargetFramework)' == 'net452'">
     <Reference Include="Microsoft.CSharp" />
     <Reference Include="System.Configuration" />
-    <Reference Include="System.Security" />
```

Comment:
Can't remove that one either. It's needed for the for the same reason as the others.

The licensing package is currently bringing in a reference:
![image](https://user-images.githubusercontent.com/753669/28996331-7cb350fa-79cc-11e7-980e-d737c37a6221.png)